src/run/run.c

```diff
@@ -1492,6 +1495,9 @@ static int start_transient_service(sd_bus *bus) {
                         if (c.memory_peak != UINT64_MAX)
                                 log_info("Memory peak: %s", FORMAT_BYTES(c.memory_peak));
 
+                        if (c.memory_swap_peak != UINT64_MAX)
+                                log_info("Memory swap peak: %s", FORMAT_BYTES(c.memory_swap_peak));
```

Comment:
I consider only showing this only if memory swap peak is greater than zero. Thoughts?

Reply:
Below, we show all statistics even if some of them are zero. I think it is ok to show zero here.